client/package.json

```diff
@@ -12,7 +12,12 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.9.0",
+    "@typescript-eslint/parser": "^5.9.0",
     "axios": "^0.21.2",
+    "eslint": "^7.25.0",
```

Comment:
A lot of these dependencies we'll probably want to move to dev dependencies. Basically if the library is required while the library/service is up and running then it would be a dependencies. Anything that is part of the development/build process would just be a dev dependency. This helps make the final product a bit smaller.

Reply:
resolved